src/mame/act/victor9k.cpp

```diff
@@ -679,28 +700,30 @@ void victor9k_state::machine_reset()
 void victor9k_state::victor9k(machine_config &config)
 {
 	// basic machine hardware
-	I8088(config, m_maincpu, XTAL(30'000'000)/6);
+	I8088(config, m_maincpu, 15_MHz_XTAL / 3);
 	m_maincpu->set_addrmap(AS_PROGRAM, &victor9k_state::victor9k_mem);
 	m_maincpu->set_irq_acknowledge_callback(I8259A_TAG, FUNC(pic8259_device::inta_cb));
 
 	// video hardware
-	screen_device &screen(SCREEN(config, SCREEN_TAG, SCREEN_TYPE_RASTER));
-	screen.set_color(rgb_t::green());
-	screen.set_refresh_hz(50);
-	screen.set_vblank_time(ATTOSECONDS_IN_USEC(2500)); // not accurate
-	screen.set_screen_update(HD46505S_TAG, FUNC(hd6845s_device::screen_update));
-	screen.set_size(640, 480);
-	screen.set_visarea(0, 640-1, 0, 480-1);
+	SCREEN(config, m_screen, SCREEN_TYPE_RASTER);
+ 	m_screen->set_color(rgb_t::green());
+ 	m_screen->set_refresh_hz(72);
+ 	m_screen->set_vblank_time(ATTOSECONDS_IN_USEC(1200)); 
+ 	m_screen->set_size(800, 400);
+ 	m_screen->set_visarea(0, 799, 0, 399);
```

Comment:
This won't work, it will cause the 6845 to be in an undefined state until it consolidates screen parameters and possibly lead to bugs. 
Since MAME doesn't support sync disable for screens it's suggested to use known CRTC setup values with `set_raw` to counteract this.

Reply:
The previous comments by Vas and OG told him explicitly not to use set_raw.  The IBM PC also starts the 6845 up in an unknown state and it works fine.